src/components/Resource/Wizard/ModelWizard.tsx

```diff
@@ -52,17 +53,27 @@ export function ModelWizard() {
   const [state, setState] = useState<WizardState>({ step: 1 });
 
   const { data: model } = trpc.model.getById.useQuery({ id: Number(id) }, { enabled: !!id });
+  console.log(model);
+  console.log(model?.modelVersions[0]?.files);
+
+  const maxSteps = 4;
 
   const editing = !!model;
   const hasVersions = model && model.modelVersions.length > 0;
-  const hasFiles = model && model.modelVersions.some((version) => version.files.length > 0);
+  const hasFiles =
+    model &&
+    model.modelVersions.some((version) =>
+      model.uploadType === ModelUploadType.Trained
+        ? version.files.filter((f) => f.type === 'Model').length > 0
```

Comment:
We used to have another file type called `Pruned Model` which there are still a couple entries in the db for them, maybe check for those as well (confirm with Justin)
```suggestion
        ? version.files.filter((f) => f.type === 'Model' || f.type === 'Pruned Model').length > 0
```

Reply:
i'll add this but I'm not sure what the distinction is between the two in database terms - a model can either be pruned or not but the "type" should still be Model imo, with maybe metadata implying that it is pruned. which i "think" i see in one of the migrations now that i look at it